dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/tab/DatasetFilesTab.java

```diff
@@ -76,11 +71,26 @@
 import java.util.Set;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import java.util.stream.Collectors;
 
+import static edu.harvard.iq.dataverse.common.BundleUtil.getStringFromBundle;
+import static edu.harvard.iq.dataverse.dataaccess.ImageThumbConverter.DEFAULT_THUMBNAIL_SIZE;
+import static edu.harvard.iq.dataverse.datacapturemodule.DataCaptureModuleUtil.rsyncSupportEnabled;
+import static edu.harvard.iq.dataverse.persistence.datafile.ExternalTool.Type.CONFIGURE;
+import static edu.harvard.iq.dataverse.persistence.datafile.ExternalTool.Type.EXPLORE;
+import static edu.harvard.iq.dataverse.persistence.datafile.ExternalTool.Type.PREVIEW;
+import static edu.harvard.iq.dataverse.persistence.dataset.DatasetLock.Reason.DcmUpload;
+import static edu.harvard.iq.dataverse.search.SearchServiceBean.SortOrder.asc;
+import static edu.harvard.iq.dataverse.settings.SettingsServiceBean.Key.DefaultDateFormat;
+import static edu.harvard.iq.dataverse.settings.SettingsServiceBean.Key.UploadMethods;
+import static edu.harvard.iq.dataverse.util.JsfHelper.addErrorMessage;
+import static edu.harvard.iq.dataverse.util.JsfHelper.addFlashErrorMessage;
+import static edu.harvard.iq.dataverse.util.JsfHelper.addFlashSuccessMessage;
+import static java.util.Collections.emptyList;
 import static java.util.stream.Collectors.joining;
 import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 
+@SuppressWarnings("serial")
```

Comment:
Why suppress those warnings?

Reply:
cause we don't care about this warning
alternatively I can add 
private static final long serialVersionUID = 1L;
instead